src/private/ptzExtension.ts

```diff
@@ -0,0 +1,296 @@
+    }
+    if (!callback) {
+      throw new Error('[ptzExtension.sendRemotePtzCustomCommand] Callback cannot be null');
+    }
+    ensureInitialized();
+    const messageId = sendMessageRequestToParent('ptzExtension.sendRemotePtzCustomCommand', [ptzControlCommand]);
+    GlobalVars.callbacks[messageId] = callback;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Terminate the remote PTZ session
+   * @param callback Callback to invoke when the command response returns.
+   */
+  export function terminateRemoteSession(callback: (error: SdkError) => void): void {
+    if (!callback) {
+      throw new Error('[ptzExtension.terminateRemotePtzSession] Callback cannot be null');
+    }
+    ensureInitialized();
+    const messageId = sendMessageRequestToParent('ptzExtension.terminateRemotePtzSession');
+    GlobalVars.callbacks[messageId] = callback;
+  }
+
+  /**
+   * Registers a handler for change in PTZ-capable participants.
+   * Only one handler can be registered at a time. A subsequent registration replaces an existing registration.
+   * @param handler The handler to invoke when the list of PTZ-capable participants changes.
+   */
+  export function registerOnCapableParticipantsChangeHandler(
+    handler: (participantChange: PtzCapableParticipants) => void,
+  ): void {
+    ensureInitialized();
+    GlobalVars.ptzCapableParticipantsChangeHandler = handler;
+    handler && sendMessageRequestToParent('registerHandler', ['ptzCapableParticipantsChange']);
+  }
+
+  /**
+   * Registers a handler for error in the PTZ handler.
+   * Only one handler can be registered at a time. A subsequent registration replaces an existing registration.
+   * @param handler The handler to invoke when there is an error from the PTZ side.
+   */
+  export function registerOnPtzErrorHandler(handler: (error: PtzHandlerFailed) => void): void {
+    ensureInitialized();
+    GlobalVars.ptzErrorHandler = handler;
+    handler && sendMessageRequestToParent('registerHandler', ['ptzHandlerError']);
+  }
+
+  /**
+   * Registers a handler for PTZ device state change.
+   * Only one handler can be registered at a time. A subsequent registration replaces an existing registration.
+   * @param handler The handler to invoke when the controlled PTZ device changes state.
+   */
+  export function registerOnControlDeviceStateChangeHandler(
+    handler: (deviceStateChange: PtzRemoteControlDeviceStateChanged) => void,
+  ): void {
+    ensureInitialized();
+    GlobalVars.ptzControlDeviceStateChangeHandler = handler;
+    handler && sendMessageRequestToParent('registerHandler', ['ptzControlDeviceStateChange']);
+  }
+
+  /**
+   * Registers a handler for PTZ session status change.
+   * Only one handler can be registered at a time. A subsequent registration replaces an existing registration.
+   * @param handler The handler to invoke when the current PTZ status changes.
```

Comment:
@param handler The handler to invoke when the current PTZ **session** status changes. ?